src/webui/www/private/preferences_content.html

```diff
@@ -731,6 +731,12 @@
             <div class="formRow" style="padding-left: 30px; padding-top: 5px;">
                 <textarea id="bypass_auth_subnet_whitelist_textarea" rows="5" cols="48" placeholder="Example: 172.17.32.0/24, fdff:ffff:c8::/40"></textarea>
             </div>
+            <table>
+                <tr>
+                    <td><label for="webUISessionTimeoutInput">QBT_TR(Session timeout:)QBT_TR[CONTEXT=OptionsDialog]</label></td>
```

Comment:
Don't forget to update WebUI translations.

Reply:
This is supposed to be sync from Transifex isn't?

Reply:
WebUI translations are also on Transifex?
So we need to remind @sledgehammer999 don't forget to update them there from time to time.

Reply:
>WebUI translations are also on Transifex?

Seems so: https://github.com/qbittorrent/qBittorrent/blob/master/.tx/config#L20